### ISSUE TITLE
print lines

### ISSUE THREAD

**Issue by GedToon** [vishnumaiea/Arduino-Snippets#1]
You should not use print lines i an interrupts service routine

**Comment by vishnumaiea** [comment]
Yes, it is not recommended to do so. But in my case, it was working and useful for debugging.
